EYBlockchain/nightfall_3
# Move to embedded SHA hash - improved performance

This PR upgrades zokrates-worker to 0.6.4 to use the re-added embedded SHA hash, which runs much faster than Zokrates' native one. Compile, setup, and generate proof all run faster.

`npm test` takes 10mins on my machine vs 30mins previously.

To test:
Build everything as normal, but make sure zokrates-worker is built fresh (`--no-cache`) and you re-setup the circuits (as they use the new hash). EDIT: Also, make sure not to use stub circuits.
`npm test` should now run faster!
